README.md

```diff
@@ -134,6 +134,8 @@ jsdom.env(config);
 - `config.done`, `config.onload`, `config.created`: see below.
 - `config.concurrentNodeIterators`: the maximum amount of `NodeIterator`s that you can use at the same time. The default is `10`; setting this to a high value will hurt performance.
 - `config.virtualConsole`: a virtual console instance that can capture the window’s console output; see the "Capturing Console Output" examples.
+- `config.pool`: an object describing which agents to use for the requests, defaults to {maxSockets: 6}, see [request module](https://github.com/request/request#requestoptions-callback) for more details.
```

Comment:
Both here and on the next line:

s/requests, defaults/requests; defaults/

Put spaces after { and before }, and make the default options `code`.